### ISSUE TITLE
Add tab content to the example

### ISSUE THREAD

**Issue by donv** [Polymer/paper-tabs#12]
The example displays how to create the tab headers, but omits displaying actual tab content.  Switching headers is good, but displaying content is better :)

**Comment by vzhen** [comment]
@frankiefu Big thanks

**Comment by AlexanderKoehn** [comment]
Just a note: For Polymer >= 1.0 the event to listen to is 'iron-select' instead of 'core-select'
`document.querySelector('paper-tabs').addEventListener('iron-select', function() {
      console.log(tabs.selected);
});`

**Comment by crclayton** [comment]
I agree, it was more complicated than it needed to be to bind pages to tabs. The change to Polymer 1+ made it all the more frustrating. 

[Here's a complete (and current) example for anyone who comes along after](http://stackoverflow.com/questions/24688986/binding-paper-tabs-to-core-pages-with-polymer/33266798#33266798)
